addon/components/frost-checkbox.js

```diff
@@ -101,7 +100,7 @@ export default Component.extend(PropTypeMixin, {
 
     input () {
       let id = this.get('value')
-      if (_.isFunction(this.attrs['onInput'])) {
+      if (typeOf(this.attrs['onInput']) === 'function') {
```

Comment:
We can start dropping `attrs` in these cases - `this.onInput` works equally well (and is nicer code)